### PR TITLE
Merge draco-7_10_0 back to develop

### PR DESCRIPTION
**Pull request by KineticTheory**
### Background

* Draco-7_10_0 released.
  * https://rtt.lanl.gov/redmine/projects/draco/wiki/Draco-7_10_0

### Status

* Reference:
  * [Pre-Merge Code Review](https://github.com/lanl/Draco/wiki/Style-Guide)
  * [CDash Status](https://rtt.lanl.gov/cdash/index.php?project=Draco&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=-pr)
* Checks
  * [x] Travis/Appveyor CI checks pass
  * [x] Code coverage does not decrease
  * [x] [Valgrind test passes](https://rtt.lanl.gov/cdash3/index.php?project=Draco)
  * [ ] [Toss3 checks pass](https://rtt.lanl.gov/cdash3/index.php?project=Draco)
  * [x] [Trinitite checks pass](https://rtt.lanl.gov/cdash3/index.php?project=Draco)
  * [ ] Code reviewed/approved, sufficient DbC checks, testing, documentation

### CODE REVIEW

File: tools/ats2-env.sh [hunk 1]
```diff
@@ -86,7 +86,7 @@ case $ddir in
       unset CPATH
       unset LD_LIBRARY_PATH
       unset LIBRARY_PATH
-      run "module load draco/xl2020.08.19-cuda-11.0.2"
+      run "module load draco/xl2020.11.12-cuda-11.2.0-beta"
```
Comment (jhchang-lanl) [inline tools/ats2-env.sh:89]: @KineticTheory Eventually we will remove the beta?

Comment (alexrlongne) [reply tools/ats2-env.sh:89]: This beta gave errors in the volta builds, can we stick with 11.0.2?

Comment (KineticTheory) [reply tools/ats2-env.sh:89]: Yes.  LLNL is rapidly incrementing the recommended toolchains.  Hopefully this will stabilize soon.

Comment (KineticTheory) [reply tools/ats2-env.sh:89]: > 
> 
> This beta gave errors in the volta builds, can we stick with 11.0.2?

We can stick with 11.0.2.  This content of this script doesn't really matter except for my release testing.
Also:
* As Jae mentioned, 11.2.0 is also available on ATS-2.
* And, the preferred/default compiler is now xl/2021.03.11.
* I'll bump the environments to the new toolchain in a couple of weeks.

Comment (alexrlongne) [reply tools/ats2-env.sh:89]: I'll test the regressions with 11.2.0 right now to see if they fixed the issues in the beta.

**Pull request merged**